modules/hal_tdk/Kconfig

```diff
@@ -17,4 +17,10 @@ config USE_EMD_ICM42670
 	depends on DT_HAS_INVENSENSE_ICM42670P_ENABLED \
 		|| DT_HAS_INVENSENSE_ICM42670S_ENABLED
 
+config USE_EMD_ICM42370
+	bool "ICM42370P High Performance 3-Axis MotionTracking Accelerometer"
```

Comment:
Add `default y` similar to `USE_EMD_ICM42670`

Reply:
Thanks for pointing me this, this is updated